Description: Change installation prefix to /usr
 .
 qhttpserver (0.1.0-0ubuntu1) precise; urgency=low
 .
   * Initial release
Author: Jeremy Stephens <jeremy.f.stephens@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- qhttpserver-0.1.0.orig/qhttpserver.pri
+++ qhttpserver-0.1.0/qhttpserver.pri
@@ -1,3 +1,3 @@
-isEmpty(PREFIX):PREFIX = /usr/local
+isEmpty(PREFIX):PREFIX = /usr
 isEmpty(LIBDIR):LIBDIR = $${PREFIX}/lib
 isEmpty(INCLUDEDIR):INCLUDEDIR = $${PREFIX}/include
